onnx/model_container.py

```diff
@@ -86,6 +86,16 @@ def check_model(self):
         if self.model_proto is not None:
             onnx.checker.check_model(self.model_proto)
 
+    def __getitem__(self, name: str) -> str | np.ndarray:
```

Comment:
Curious when we will return a string?

Reply:
It cannot. I misread an annotation above.